tests/foreman/ui/test_contentaccess.py

```diff
@@ -146,6 +146,33 @@ def setUpClass(cls):
         cls.activation_key = entities.ActivationKey(
             organization=cls.session_org).create()
 
+    def _setup_virtual_machine(self, vm):
```

Comment:
Agreed with @oshtaier can't we put this function on a common `helpers` file and import on both places instead of repeating?

Reply:
https://github.com/SatelliteQE/robottelo/blob/master/robottelo/helpers.py

Reply:
@rochacbruno if we put that function in a helper function we have to pass to it all the params repos repos_info org_label custom_product_label that params can be very confusing for a general usage purpose, and is very understood in the context of the test cases.
I implemented it this way as a compromise , but if everybody insist on that , I will do it the way everybody want it. But I am sure that helper function will bring more confusion than clarity.

Reply:
@ldjebran @oshtaier what about a method on base `TestCase` as in https://github.com/SatelliteQE/robottelo/blob/master/robottelo/test.py#L206